.github/workflows/support/citr/checkValidatorNode.sh

```diff
@@ -0,0 +1,26 @@
+node_id=$1
+
+ps -aef | grep -w stateAnalyzer | grep -v grep | grep stateAnalyzer >/dev/null
+if [ $? -eq 0 ]
```

Comment:
```suggestion
if [[ ${?} -eq 0 ]]
```

Should opt for bash style conditionals since bash is the default shell environment for our github runners.